apps/docs/src/guide/getting-started/react-example.md

```diff
@@ -0,0 +1,35 @@
+# React Example
+
+<!-- TODO: Create properly code snippet on new package: `app/react-app` after https://github.com/FuelLabs/fuels-ts/pull/827 got merged -->
```

Comment:
Is this comment correct? That PR looks very old

Reply:
I just moved it as it was, but it's probably obsolete.